RPIPs/RPIP-43.md

```diff
@@ -57,13 +57,13 @@ megapools SHALL include at least the following:
 Node operators can manage the set of validators in their megapool:
 
 - A Node Operator SHALL be able to add new validators to their megapool
-- A Node Operator MAY be able to migrate existing validators to their megapool
+- A Node Operator MAY! be able to migrate existing validators to their megapool
      - Note that this would involve updating the validators' BLS withdrawal
        credentials to the megapool address
 - New and existing validators' withdrawal credentials MUST be checked to
   correctly refer to the megapool address, with the validator status set to
   Dissolved if the check fails
-     - The check MAY occur after the validator is added initially as Prestaked
+     - The check MAY? occur after the validator is added initially as Prestaked
```

Comment:
I think this is fine? I'd argue the whole clause is kind of unnecessary, but perhaps I'm missing something.

Reply:
Yeah, I'm confused why this clause is in there as well. Seems like the check hast to occur after but that's not really something we need to spec, because doing it before just isn't possible?